partition_processor.go

```diff
@@ -528,7 +546,45 @@ func (pp *PartitionProcessor) enqueueTrackOutputStats(ctx context.Context, topic
 	})
 }
 
-func (pp *PartitionProcessor) processMessage(ctx context.Context, wg *sync.WaitGroup, msg *sarama.ConsumerMessage, syncFailer func(err error), asyncFailer func(err error)) error {
+func (pp *PartitionProcessor) processVisit(ctx context.Context, wg *sync.WaitGroup, v *visit, syncFailer func(err error), asyncFailer func(err error)) error {
+	cb, ok := pp.visitCallbacks[v.name]
+	// no callback registered for migration
+	if !ok {
+		return fmt.Errorf("no callback registered for migration")
+	}
+
```

Comment:
fixed, this was just legacy since the feature was earlier called "migration" based on our main use case. Just later we decided to rename it to "visit" as it's more generic.